From 8507771ad72d0f4e69bab8c6234364ebaf5e06ff Mon Sep 17 00:00:00 2001
From: Kevin Mihelich <kevin@archlinuxarm.org>
Date: Tue, 13 Feb 2018 21:27:47 -0700
Subject: [PATCH] ARM: Samsung: Add Exynos5422-based Odroid HC1+ support

Updated HC1 ADC value to HC1+ to capture both revisions.

Signed-off-by: Kevin Mihelich <kevin@archlinuxarm.org>
---
 board/samsung/common/exynos5-dt-types.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/board/samsung/common/exynos5-dt-types.c b/board/samsung/common/exynos5-dt-types.c
index 03d3a3112a..8c92ea3e9b 100644
--- a/board/samsung/common/exynos5-dt-types.c
+++ b/board/samsung/common/exynos5-dt-types.c
@@ -32,7 +32,8 @@ static const struct udevice_id board_ids[] = {
  * 0.3  1280     XU4 0.1
  * 0.4   739     XU4 0.2
  * 0.5  1016     XU4+Air0.1 (Passive cooling)
- * 0.6  1308     XU4S 0.1 (HC1)
+ * 0.6  1308     XU4-HC1 0.1
+ * 0.7  1470     XU4-HC1+ 0.1
  * Use +1% for ADC value tolerance in the array below, the code loops until
  * the measured ADC value is lower than then ADCmax from the array.
  */
@@ -40,7 +41,7 @@ struct odroid_rev_info odroid_info[] = {
 	{ EXYNOS5_BOARD_ODROID_XU3_REV01, 1, 10, "xu3" },
 	{ EXYNOS5_BOARD_ODROID_XU3_REV02, 2, 375, "xu3" },
 	{ EXYNOS5_BOARD_ODROID_XU4_REV01, 1, 1293, "xu4" },
-	{ EXYNOS5_BOARD_ODROID_HC1_REV01, 1, 1321, "hc1" },
+	{ EXYNOS5_BOARD_ODROID_HC1_REV01, 1, 1485, "hc1" },
 	{ EXYNOS5_BOARD_ODROID_UNKNOWN, 0, 4095, "unknown" },
 };
 
-- 
2.16.1

